packages/react-button/src/components/CompoundButton/useCompoundButtonStyles.ts

```diff
@@ -1,356 +1,224 @@
 import { mergeClasses, makeStyles } from '@fluentui/react-make-styles';
-import { Theme } from '@fluentui/react-theme';
 import { buttonSpacing, useButtonStyles } from '../Button/useButtonStyles';
-import { CompoundButtonState, CompoundButtonStyleSelectors, CompoundButtonVariantTokens } from './CompoundButton.types';
+import { CompoundButtonState } from './CompoundButton.types';
 
 const CompoundButtonClassNames = {
   secondaryContent: 'CompoundButton-secondaryContent',
 };
 
-export const makeCompoundButtonTokens = (theme: Theme): CompoundButtonVariantTokens => ({
-  base: {
-    // root tokens
-    height: 'auto',
-    paddingX: buttonSpacing.large,
-    paddingY: buttonSpacing.large,
+const useRootStyles = makeStyles({
+  base: theme => ({
+    gap: buttonSpacing.large,
```

Comment:
Does `gap` need a similar TODO comment as in useButtonStyles?

Reply:
Yes, let me add it.